setup.py

```diff
@@ -26,7 +26,8 @@
     description="Google Drive API made easy. Maintained fork of PyDrive.",
     long_description=open("README.rst").read(),
     install_requires=[
-        "google-api-python-client >= 1.7.12",
+        "google-api-python-client >= 1.12.1",
+        "six >= 1.13.0",
```

Comment:
Need to update this as well, don't see a better way. We might need to do the same in `dvc`.

Reply:
hmmm ... I don't understand why is it needed, google-api-python-client already specifies the same dependency. Any ideas on this?

Reply:
`pip` does not have a resolver (yet), so it either installs top-level requirements specification (as done here) or does "first one wins" which installs `six` from different dependency and ignores the one specified in `google-api-python-client` (it does throw warning at the end). 
https://github.com/pypa/pip/issues/988#issue-15376371

Reply:
kk, thanks @skshetry!